From 56fb30d93e96d7f983af0cb4dff9155da91dd671 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Mon, 25 Feb 2019 16:37:19 +0800
Subject: socket-util.h: include string.h

Fix the following compile error:
src/basic/socket-util.h:187:30: error: implicit declaration of function 'strnlen'; did you mean 'strlen'? [-Werror=implicit-function-declaration]

Upstream-Status: Accepted

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/basic/socket-util.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/socket-util.h b/src/basic/socket-util.h
index 6920fd99ba..36ec422f2c 100644
--- a/src/basic/socket-util.h
+++ b/src/basic/socket-util.h
@@ -9,6 +9,7 @@
 #include <netinet/in.h>
 #include <stdbool.h>
 #include <stddef.h>
+#include <string.h>
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <sys/un.h>
-- 
2.20.1

